src/lib/storage/constraints/table_order_constraint.cpp

```diff
@@ -0,0 +1,40 @@
+#include "table_order_constraint.hpp"
+
+#include <boost/container_hash/hash.hpp>
+
+namespace hyrise {
+
+TableOrderConstraint::TableOrderConstraint(const std::vector<ColumnID>& ordering_columns,
+                                           const std::vector<ColumnID>& ordered_columns)
+    : _ordering_columns(ordering_columns), _ordered_columns(ordered_columns) {}
```

Comment:
Just curious, there is nothing we need to check here? Right order etc.?

Reply:
We (could) have a `(Debug)Assert` checking that the columns do not intersect (which we could also check for key constraints). Besides that, no restrictions.